.github/workflows/tests.yml

```diff
@@ -256,10 +256,10 @@ jobs:
           USER_NAME: ${{ secrets.USER_NAME }}
           ACCESS_TOKEN: ${{ secrets.ACCESS_TOKEN }}
         run: |
-          mvn -s $GITHUB_WORKSPACE/.github/workflows/settings.xml -B -V -e -Pcompose -Dmicroservice-docker -Dquickstart-docker -Ddeploy -Dtar -DskipTests clean install
+          mvn -s $GITHUB_WORKSPACE/.github/workflows/settings.xml -B -V -e -Pcompose -Dmicroservice-docker -Dquickstart-docker -Dquickstart-maven -Ddeploy -Dtar -DskipTests -Dmaven.build.cache.enabled=false clean install
```

Comment:
This is almost done, but it's actually still using the download URL.  In order to guarantee that it is pulling the tarballs from maven, you need to set bogus URLs for the various tarballs.

e.g.
```
DW_ZOOKEEPER_DIST_URI='http://some/bogus/path/zookeeper.tar.gz'
DW_ACCUMULO_DIST_URI='http://some/bogus/path/accumulo.tar.gz'
DW_WILDFLY_DIST_URI='http://some/bogus/path/wildfly.tar.gz'
DW_HADOOP_DIST_URI='http://some/bogus/path/hadoop.tar.gz'
DW_MAVEN_DIST_URI='http://some/bogus/path/maven.tar.gz'
```